lisp/mastodon-profile.el

```diff
@@ -0,0 +1,154 @@
+  (mastodon-tl--property 'toot-json))
+
+(defun mastodon-profile--make-author-buffer (status)
+  "Take a STATUS and inserts a user account into a new buffer."
+  (let* ((id (mastodon-profile--field status 'id))
+         (acct (mastodon-profile--field status 'acct))
+         accounts/id/statuses
+         (url (mastodon-http--api
+               (concat "accounts/"
+                       (format "%s" id)
+                       "/statuses" )))
+         (buffer (concat "*mastodon-" acct  "*"))
+         (account (cdr(assoc 'account status)))
+         (note (mastodon-profile--field status 'note))
+         (json (mastodon-http--get-json url)))
+    (with-output-to-temp-buffer buffer
+      (switch-to-buffer buffer)
+      (mastodon-mode)
+      (setq mastodon-tl--buffer-spec
+            `(buffer-name ,buffer
+                          endpoint ,(format "accounts/%s/statuses" id)
+                          update-function
+                          ,'mastodon-tl--timeline json))
+      (let ((inhibit-read-only t))
+        "\n"
+        (mastodon-profile--image-from-status account)
+        "\n"
+        (propertize (mastodon-profile--field status 'display_name
+                                             'face 'mastodon-display-name-face))
+        "\n"
+        (propertize acct
+                    'face 'default)
+        "\n ------------\n"
+        (mastodon-tl--render-text note status)
+        (mastodon-tl--set-face
+         (concat " ------------\n"
+                 "     TOOTS   \n"
+                 " ------------\n")
+         'success)
+        (mastodon-tl--timeline json)))
+    (mastodon-tl--goto-next-toot)))
+
+(defun mastodon-profile--get-next-author ()
+  "Jump to the next author and generate a buffer."
+  (interactive)
+  (mastodon-profile--make-author-buffer (mastodon-profile--toot-json)))
+
+(defun mastodon-profile--image-from-status (status)
+  "Generate an image from a STATUS."
+  (let ((url (cdr(assoc 'avatar_static status))))
+    (unless (equal url "/avatars/original/missing.png")
+      (mastodon-media--get-media-link-rendering url))))
+
+(defun mastodon-profile--field (status field)
+  "The STATUS is a nested alist.
+
+FIELD is used to identify regions under 'account"
+  (cdr (assoc field (assoc 'account status))))
+
+(defun mastodon-profile--get-next-authour-id ()
+  "Get the author id of the next toot."
+  (interactive)
+  (get-authour-id (toot-proporties)))
+
+(defun mastodon-profile--add-author-bylines (tootv)
+  "Convert TOOTV into a author-bylines and insert."
+  (let ((inhibit-read-only t))
+    (mapc (lambda(toot)
+            (insert (propertize (mastodon-tl--byline-author
+                                 (list (append (list 'account) toot)))
+                                'byline  't
+                                'toot-id (cdr(assoc 'id toot)) 'toot-json toot)
+                    "\n"))
+          tootv))
+  (mastodon-media--inline-images))
+
+
+(defun mastodon-profile--get-following ()
```

Comment:
This and `mastodon-profile--followers` aren't hooked up yet?

Reply:
They work, but i could not think of an intuitive key binding.

We can brainstorm and give them a keybinding before the 0.8.0 release.

Reply:
Sounds good.